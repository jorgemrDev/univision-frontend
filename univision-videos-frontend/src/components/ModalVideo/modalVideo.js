import React, { useState } from "react";
import { Modal } from "antd";
import ReactPlayer from "react-player";
import "./modalVideo.scss";

export default function ModalVideo(props) {
  const { videoPath, isOpen, close } = props;
  const [urlVideo, setUrlVideo] = useState(videoPath);

  return (
    <Modal
      className="modal-video"
      visible={isOpen}
      centered
      onCancel={close}
      footer={false}
    >
      <ReactPlayer url={urlVideo} controls></ReactPlayer>
    </Modal>
  );
}
