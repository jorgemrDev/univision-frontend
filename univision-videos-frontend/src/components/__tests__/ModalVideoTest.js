import React from "react";
import ModalVideo from "../ModalVideo";
import renderer from "react-test-renderer";

const props = {
  videoPath: "videoPath",
  isOpen: false,
  close: function () {},
};

test("Render ModalVideo correctly", () => {
  const component = renderer.create(
    <ModalVideo
      videoPath={props.videoPath}
      isOpen={props.isOpen}
      close={props.close}
    ></ModalVideo>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
