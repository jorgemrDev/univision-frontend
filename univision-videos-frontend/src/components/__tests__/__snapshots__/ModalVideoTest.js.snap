// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render ModalVideo correctly 1`] = `null`;
